components/hab/mac/plan.sh

```diff
@@ -7,10 +7,6 @@ pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_deps=()
 pkg_build_deps=()
 
-nproc() {
-  sysctl -n hw.ncpu
-}
```

Comment:
I had to go back to remember why I added this. Answer: initially we had `cargo -j $(nproc) … build` and I removed this in bf5724191f3d94c70388040a9fb8331bcb6e1bd8#diff-53cf941dbc3096011996a1427420a6e4 for more consistency (I think that Cargo does this by default anyway these days). So, whoops!